eng/Version.Details.xml

```diff
@@ -84,15 +84,15 @@
     -->
     <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="3.1.8">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>f375e037fec56a07f0cb8cde2c1ac60e8e1e5bb2</Sha>
+      <Sha>f96e1b909b3668c773f4d2f8c16a0633c8335409</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.1.6">
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup</Uri>
       <Sha>3acd9b0cd16596bad450c82be08780875a73c05c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="3.1.8-servicing.20415.1">
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="3.1.8-servicing.20418.1">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>f375e037fec56a07f0cb8cde2c1ac60e8e1e5bb2</Sha>
+      <Sha>f96e1b909b3668c773f4d2f8c16a0633c8335409</Sha>
```

Comment:
Current core-setup SHA is dotnet/core-setup@08b4abfbf917229c9a23d474d1eabe2b34735343 (`08b4abfbf917229c9a23d474d1eabe2b34735343` in full). That's building in https://dev.azure.com/dnceng/internal/_build/results?buildId=776913&view=results and contains @mmitche's dotnet/core-setup#9069 fix. Look for another commit on this branch in about 2 hours (given current agent wait times).